src/features/VisualNovel/pages/VisualNovelPage/VisualNovelPage.module.css

```diff
@@ -1,38 +1,40 @@
 .page {
-  padding-top: 32px;
+  padding-top: var(--chakra-sizes-8);
   display: flex;
   flex-direction: column;
-  gap: 16px;
+  gap: var(--chakra-sizes-4);
   overflow: hidden;
 }
 
 .header {
   display: flex;
   flex-direction: column;
-  gap: 16px;
+  gap: var(--chakra-sizes-4);
 }
 
 .overview {
-  padding: 0 40px;
+  padding: 0 var(--chakra-sizes-10);
   display: grid;
-  grid-template-columns: 200px 1fr;
-  gap: 32px;
+  grid-template-columns: var(--chakra-sizes-48) 1fr;
+  gap: var(--chakra-sizes-8);
 }
 
 .info {
   display: flex;
   flex-direction: column;
-  gap: 24px;
+  gap: var(--chakra-sizes-6);
 }
 
 .image {
-  height: 300px;
```

Comment:
Is this intentional? We are breaking image style with this `height: auto`

Reply:
I add this so the image have proper aspect ratio. Also I made boundary if the image is too tall.
I think we would need to set height for vn header block, when implementing "read more" functionality for description.